testdata/cheats/Fork2.t.sol

```diff
@@ -165,6 +165,94 @@ contract ForkTest is DSTest {
         // this will revert since `dummy` does not exists on the currently active fork
         string memory msg2 = dummy.hello();
     }
+
+    function testGetLogs() public {
+        cheats.selectFork(mainnetFork);
+        address weth = address(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2);
+        bytes32 withdrawalTopic = 0x7fcf532c15f0a6db0bd6d0e038bea71d30d808c7d98cb3bf7268a95bf5081b65;
+        uint256 blockNumber = 17623835;
+
+        // getLogs call
+        // {
+        //     "jsonrpc": "2.0",
+        //     "method": "eth_getLogs",
+        //     "id": "1",
+        //     "params": [
+        //         {
+        //             "address": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        //             "fromBlock": "0x10CEB1B",
+        //             "toBlock": "0x10CEB1B",
+        //             "topics": [
+        //                 "0x7fcf532c15f0a6db0bd6d0e038bea71d30d808c7d98cb3bf7268a95bf5081b65"
+        //             ]
+        //         }
+        //     ]
+        // }
+
+        bytes32[] memory topics = new bytes32[](1);
+        topics[0] = withdrawalTopic;
+
+        Cheats.Log[] memory logs = cheats.getLogs(blockNumber, blockNumber, weth, topics);
+        assertEq(logs.length, 3);
+
+        // check that the logs are correct
+        for (uint256 i = 0; i < logs.length; i++) {
+            Cheats.Log memory log = logs[i];
+            assertEq(log.emitter, weth);
+            assertEq(log.topics.length, 2);
+            assertEq(log.topics[0], withdrawalTopic);
+            // TODO check that log.topics[1] and log.data match the response once the
+            // response is fixture-ized
+        }
+
+        /* The response from the above eth_call
+        // TODO put into a fixture
```

Comment:
sweet yes! i'd also include some comments on how to regen this fixture, if needed.

Reply:
Included in `README.md` under `testdata/fixtures/Rpc` folder.